### ISSUE TITLE
How to use convert a number?

### ISSUE THREAD

**Issue by WG-** [josdejong/mathjs#1206]
As a first contribution to the project, I thought I would implement `rad2deg` and `deg2rad`. However, then I found out mathjs has a unit module. But how does that exactly work? Let say I have a variable `x`, which I know is in radians, did not explicitly specify it, but I want to convert it to degrees? How can I achieve this? Is there a oneliner? Something like `f.unit.to('deg','rad')`? In addition, does the result also store the unit in the new variable? Because frankly I do not care about it and most likely it makes all operations after slower, because it takes the unit into account when doing calculations?

**Comment by josdejong** [comment]
In the expression parser you could do `x * rad / deg` or create a function yourself like 

```
rad2deg(x) = x * rad / deg
rad2deg(2 * pi) # 360
```
The equivalent in plain javascript is:
```js
function radToDeg(x) { 
  return math.divide(math.multiply(x, math.unit('rad')), math.unit('deg')) 
}
```

**Comment by WG-** [comment]
Okay, but we don't want a dedicated function for this? Such that you can simply call `math.radToDeg` or `math.degToRad`?

**Comment by josdejong** [comment]
Yeah, if there is need for it we can consider it. So far you're the first asking about it :). It's indeed quite a common util function for working with angles. 

The idea of using units instead of converting numbers yourself is that you can simply do `sin(1.57 rad)` and `sin(90 deg)` so you cannot accidentally make mistakes with units. These sort of issues have literally crashed rockets in the past, it's a dream of mine to get rid of issues these type of issues by offering a system where you can work with units instead of numbers.

**Comment by WG-** [comment]
You are totally correct that unit issues made rockets crash in the past. But I would prefer to have a simple accesible function for that. I think as well that it is a common util function, which I normally use quiet often in the field of control engineering and digital signal processing. If it is okay I will create those two functions and submit a PR.

**Comment by josdejong** [comment]
Yes I'm ok with that. I agree that its a very common/handy util function.
